bitnami/kibana/templates/service.yaml

```
@@ -3,6 +3,9 @@ kind: Service
 metadata:
   name: {{ include "common.names.fullname" . }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
+{{- if .Values.service.labels }}
+{{ toYaml .Values.service.labels | indent 4}}
+{{- end }}
```

Please user the "common.tplvalues.render" named template instead: 
```suggestion
  {{- if .Values.service.labels }}
  {{- include "common.tplvalues.render" (dict "value" .Values.service.labels "context" $) | nindent 4 }}
  {{- end }}
```

---

Thanks for the suggestion! Already committed the changes.